core/src/main/java/org/apache/carbondata/core/statusmanager/SegmentStatus.java

```diff
@@ -58,7 +58,7 @@
   /**
    * Segment is compacted
    */
-  @SerializedName("Compacted")
+  @SerializedName(value = "C", alternate = "Compacted")
```

Comment:
why only for Success and Compacted? what about the Marked for delete, Partial Success, Insert In Progress and other status

Reply:
In table status file, most of the entries are SUCCESS and COMPACTED, so I gave alias for these two only, for others I prefer to keep it the same for better readibility

Reply:
yes you are right, but i think since goal is to reduce size, better to change all the status, because when we have both status, like one with short form, one with full name do not look good, what do you think?